app/graphql/types/customer_portal_mutation_type.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module Types
+  class CustomerPortalMutationType < Types::BaseObject
```

Comment:
should we namespace it instead? (some fore query, etc)

```suggestion
  module CustomerPortal
    class MutationType < Types::BaseObject
```